tracing/span.go

```diff
@@ -18,6 +18,20 @@ const (
 	SpanTypeServer
 )
 
+// String returns a string representing the SpanType.
```

Comment:
🔕 golint actually don't force you to add doc comment for `type.String() string` (also `type.Error() string`) as that's part of the builtin type magic.